src/command.c

```diff
@@ -177,7 +177,7 @@ bool cmd_version(target_s *t, int argc, const char **argv)
 #else
 	gdb_out(BOARD_IDENT);
 	gdb_outf(", Hardware Version %d\n", platform_hwversion());
-	gdb_out("Copyright (C) 2022 Black Magic Debug Project\n");
+	gdb_out("Copyright (C) 2023 Black Magic Debug Project\n");
```

Comment:
Particularly good catch - though this should probably state `2010-2023` or so as the project's been around way longer and one is supposed to declare a year range

Reply:
Yes, I should've gone for the proper fix instead of the quick one :) done